src/components/Icon/urls.tsx

```diff
@@ -21,3 +21,6 @@ export { default as Globe } from './assets/Globe.svg';
 export { default as CaretRight } from './assets/CaretRight.svg';
 export { default as Cross } from './assets/Cross.svg';
 export { default as Edit } from './assets/Edit.svg';
+export { default as DialogSuccess } from './assets/dialogSuccess.svg';
```

Comment:
Call them `Success`, `Warning` and `Danger` only, without the `Dialog` prefix.

Also, make sure that the files have exactly the same name as the components.